examples/resources/gcore_cdn_originshielding/resource.tf

```diff
@@ -0,0 +1,68 @@
+provider gcore {
+  permanent_api_token = "251$d3361.............1b35f26d8"
+}
+
+resource "gcore_cdn_origingroup" "origin_group_1" {
+  name     = "origin_group_1"
+  use_next = true
+  origin {
+    source  = "example.com"
+    enabled = true
+  }
+}
+
+resource "gcore_cdn_resource" "cdn_example_com" {
+  cname               = "cdn.example.com"
+  origin_group        = gcore_cdn_origingroup.origin_group_1.id
+  origin_protocol     = "MATCH"
+  secondary_hostnames = ["cdn2.example.com"]
+
+  options {
+    edge_cache_settings {
+      default = "8d"
+    }
+    browser_cache_settings {
+      value = "1d"
+    }
+    redirect_http_to_https {
+      value = true
+    }
+    request_limiter {
+      rate_unit = "r/s"
+      rate = 5
+      burst = 1
+    }
+    gzip_on {
+      value = true
+    }
+    cors {
+      value = [
+        "*"
+      ]
+    }
+    rewrite {
+      body = "/(.*) /$1"
+    }
+
+    tls_versions {
+      enabled = true
+      value = [
+        "TLSv1.2",
+      ]
+    }
+
+    force_return {
+      code = 200
+      body = "OK"
+    }
+  }
+}
+
+data "gcore_cdn_shielding_location" "sl" {
+  city = "Luxembourg"
```

Comment:
Hi! I'm quite not sure why we decided to describe shields by the city? because it's not a `unique` field, so we can have more than one shield in the same city. The unique identifier of shield is `ID` or `datacenter`

Reply:
Hi! datacenter must be better, a good point. But according to DB records duplicates by city are not public ones and for user will be easier to get it by city name.

Reply:
Fixed